src/NServiceBus.Persistence.ServiceFabric.Tests/ComponentTests/Sagas/DefaultSagaIdGenerator.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace NServiceBus.Persistence.ComponentTests
+{
+    using System;
+    using Sagas;
+
+    class DefaultSagaIdGenerator : ISagaIdGenerator
+    {
+        public Guid Generate(SagaIdGeneratorContext context)
+        {
+            return CombGuid.Generate();
```

Comment:
I copied the core default persister since it is internal. If the tests would be run against persister which assumes core behavior it might be dangerous to use `Guid.NewGuid`